README.md

```diff
@@ -24,15 +24,16 @@
 
    ```xml
    <?xml version="1.0"?>
-   <ruleset name="Webimpress Coding Standard"
+   <ruleset name="MyCodingStandard"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
             xsi:noNamespaceSchemaLocation="./vendor/squizlabs/php_codesniffer/phpcs.xsd">
-       <rule ref="./vendor/webimpress/coding-standard/ruleset.xml"/>
```

Comment:
Can you explain what's wrong with that method?

Reply:
All phpcs ruleset I've seen have a name and can be used by its name.

Reply:
Not sure if this is still relevant, but I came across this PR when looking for something else.

@michalbundyra I've previously seen issues with sniff name based configuration not always working 100% correctly when a ruleset is imported via a file ref vs by name.

I suppose I should do a deep-dive into this at some point, but I agree with the recommendation that standards containing sniffs should be reference-able by name and that the recommendation should be to register them in `installed_paths` and include them by name.

Also see my comments here:
* https://github.com/slevomat/coding-standard/issues/1413#issuecomment-1191805418
* https://github.com/slevomat/coding-standard/issues/1413#issuecomment-1191871990

Reply:
Thanks, @jrfnl ! I'll have a look and try to sort it out.